zsh-proxy.plugin.zsh

```diff
@@ -32,14 +33,15 @@ __check_ip() {
     echo "========================================"
     echo "Check what your IP is"
     echo "----------------------------------------"
-    echo -n "IPIP.NET: "
-    curl https://myip.ipip.net
+    echo -n "IPv4: "
+    curl -4 https://ifconfig.co
```

Comment:
I personally prefer `ip.sb` since I made this plugin mainly for Chinese users while `ipconfig.co` has some connectivity issue in China.

```suggestion
    curl -4 https://ipv4.ip.sb
```

Reply:
Changed with: curl -s https://api-ipv4.ip.sb/ip